python/pyarrow/conftest.py

```diff
@@ -241,3 +242,26 @@ def _docdir(request):
 
     else:
         yield
+
+
+# Define doctest_namespace for fs module docstring import
+@pytest.fixture(autouse=True)
+def add_fs(doctest_namespace, request):
+
+    # Trigger ONLY for the doctests
+    doctest_m = request.config.option.doctestmodules
+    doctest_c = getattr(request.config.option, "doctest_cython", False)
+
+    if doctest_m or doctest_c:
+        # fs import
+        doctest_namespace["fs"] = fs
+
+        # Creation of an object and file with data
+        local = fs.LocalFileSystem()
+        with local.open_output_stream('/tmp/fileinfo.dat') as stream:
```

Comment:
Hmm, it would be nice if we could avoid creating files without deleting them at the end...

Reply:
Not sure how then to show copying files etc.?

Reply:
Oh, sorry. Misread 🤦‍♀️ 
Of course, will add the deleting part +1